tests/rptest/utils/node_operations.py

```diff
@@ -330,6 +332,7 @@ def stop_node(self, idx):
         with self.lock:
             self.redpanda.remove_from_started_nodes(node)
             self.redpanda.stop_node(node)
+            time.sleep(self.wait_after_stop)
```

Comment:
how exactly this fixes the issue? The test failure is a idempotency violation, I thought that would be a code bug in most cases.

Reply:
The test uses both random failures and node decommissions. The problem was reproduced in the following way:
1. node 2 was the leader, node 3 was mostly up to date with it, and node 4 was behind
2. node 2 was restarted, and very shortly after it node 3 got down
3. node 4 became a candidate and node 2 voted for it because when 2 restarted it could not recover up to the `majority_replicated_index` point, but rather only till some point below -- is it normal?
4. so some raft data was lost